app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ingredients/IngredientsProductFragment.java

```diff
@@ -70,569 +75,675 @@
-                textAdditiveProductCardView.setVisibility(View.VISIBLE);
-                additiveProduct.append(getString(R.string.txtLoading));
-                break;
-            }
-            case EMPTY: {
-                textAdditiveProductCardView.setVisibility(View.GONE);
-                break;
-            }
-        }
-    }
-
-    @Override
-    public void showAllergens(List<AllergenName> allergens) {
-        substanceProduct.setMovementMethod(LinkMovementMethod.getInstance());
-        substanceProduct.setText(bold(getString(R.string.txtSubstances)));
-        substanceProduct.append(" ");
-
-        for (int i = 0, lastIdx = allergens.size() - 1; i <= lastIdx; i++) {
-            AllergenName allergen = allergens.get(i);
-            substanceProduct.append(getAllergensTag(allergen));
-            // Add comma if not the last item
-            if (i != lastIdx) substanceProduct.append(", ");
-        }
-    }
-
-    @Override
-    public void showAllergensState(String state) {
-        switch (state) {
-            case LOADING: {
-                textSubstanceProductCardView.setVisibility(View.VISIBLE);
-                substanceProduct.append(getString(R.string.txtLoading));
-                break;
-            }
-            case EMPTY: {
-                textSubstanceProductCardView.setVisibility(View.GONE);
-                break;
-            }
-        }
-    }
-
-    private List<String> getAllergens() {
-        List<String> allergens = mState.getProduct().getAllergensTags();
-        if (mState.getProduct() == null || allergens == null || allergens.isEmpty()) {
-            return Collections.emptyList();
-        } else {
-            return allergens;
-        }
-    }
-
-    @OnClick(R.id.imageViewIngredients)
-    public void openFullScreen(View v) {
-        if (mUrlImage != null) {
-            Intent intent = new Intent(v.getContext(), FullScreenImage.class);
-            Bundle bundle = new Bundle();
-            bundle.putString("imageurl", mUrlImage);
-            intent.putExtras(bundle);
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-                ActivityOptionsCompat options = ActivityOptionsCompat.
-                        makeSceneTransitionAnimation(getActivity(), (View) mImageIngredients,
-                                getActivity().getString(R.string.product_transition));
-                startActivity(intent, options.toBundle());
-            } else {
-                startActivity(intent);
-            }
-
-        } else {
-            // take a picture
-            if (ContextCompat.checkSelfPermission(getActivity(), CAMERA) != PERMISSION_GRANTED) {
-                ActivityCompat.requestPermissions(getActivity(), new String[]{CAMERA}, MY_PERMISSIONS_REQUEST_CAMERA);
-            } else {
-                EasyImage.openCamera(this, 0);
+public class IngredientsProductFragment extends BaseFragment implements IIngredientsProductPresenter.View
+{
```

Comment:
why is the formatting changed in this file

Reply:
You are right. I was working on a different project that is using a different formatting and I forgot to switch it back before pushing the patch. So I reformatted again and updated the mr.
Thanks.